generate.py

```diff
@@ -3,7 +3,7 @@
 
 from lsp_schema import MetaModel
 from utils.generate_enumerations import generate_enumerations
-from utils.generate_requests import generate_requests
+# from utils.generate_requests import generate_requests
```

Comment:
Generate request can be deleted,
As it is not needed now.
What do you think?

Reply:
sure, can clean up

Reply:
Thanks :⁠-⁠)